build.gradle

```diff
@@ -130,6 +130,14 @@ private void configureAndroidLibrary(Project project) {
         project.findProperty('VERSION_NAME') :
         rootProject.findProperty('VERSION_NAME')
 
+    if (project.hasProperty('signingKeyId')) {
+        System.out.println("Getting signing info from protected source.")
+        project.ext.'signing.keyId' = findProperty('signingKeyId').trim()   
+        project.ext.'signing.password' = findProperty('signingPassword').trim()
```

Comment:
Why do you need to `trim()` the value?

Reply:
Actually, we don't. I was troubleshooting an issue locally and just forgot to remove it